_config.yml

```diff
@@ -25,8 +25,21 @@ gems:
   - jekyll-assets
   - jekyll-sitemap
 
-name: cga_site
+name: cloud.gov.au
 
 permalink: pretty
 
 url: # add site url
+
+global_navigation:
+  - Overview: '/overview/'
+  - Docs: 'http://docs.cloud.gov.au'
```

Comment:
I think perhaps we may need to make it clearer to our users that this will navigate to another page